src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -694,6 +694,9 @@ compiler.err.illegal.combination.of.modifiers=\
 compiler.err.illegal.enum.static.ref=\
     illegal reference to static field from initializer
 
+compiler.err.illegal.char.literal.multiple.surrogates=\
+    character literal contains more than one UTF-16 code point
```

Comment:
The error message is kind of vague. How about using "surrogate code point"?
https://www.unicode.org/glossary/#surrogate_code_point

Reply:
Or "UTF-16 code unit"?